scalafix-core/shared/src/main/scala/scalafix/internal/rule/OptionConstructor.scala

```diff
@@ -0,0 +1,28 @@
+package scalafix.internal.rule
+
+import scala.meta._
+import scala.meta.tokens.Token.LeftBrace
+
+import scalafix.Patch
+import scalafix.rule.{Rule, RuleCtx, RuleName}
+import scalafix.util.Trivia
+
+case object OptionConstructor
+    extends Rule(
+      RuleName("OptionConstructor").withDeprecatedName(
+        "OptionConstructor",
+        "Use the Option constructor instead",
+        "0.6.0"
```

Comment:
this looks wrong, `withDeprecatedName` is used for rules that have been renamed. You don't need this since this is a new rule.